File: OOAD_HW1/Q4/rhino.java
package Q4;

public class rhino extends pachyderm {

	// Rhino's name
	private String name;
	
	// Default Constructor, don't use.
	public rhino() {
		super("None");
		this.name = "None";
	}
	
	// Constructor for setting the Rhino's name, and it's super-class' name.
	public rhino(String name) {
		super(name);
		this.name = name;
	}
}

File: OOAD_HW2/Part B/Rhino.cpp
#include <iostream>
#include <string>
#include <cstdlib>
#include <iomanip>

using namespace std;

class Rhino: public Pachyderm
{
    protected:
        string name;
        string type;
        //NoiseBehavior *noise;

    public:
        Rhino(string aName, string aType);
};

// Behavior handled by interface via super class.
Rhino::Rhino(string aName, string aType): Pachyderm(aName, aType)
{
    name = aName;
    type = aType;
}



